src/bayesian.py

```diff
@@ -0,0 +1,50 @@
+from bayes_opt import BayesianOptimization
+import numpy as np
+from typing import List, Optional
+
+
+N_grid_points = 1000
+
+# This jupyter notebook helped me
+# https://github.com/fmfn/BayesianOptimization/blob/master/examples/visualization.ipynb
+
+
+def run_one_1d_bayesian_optimization(data: List[dict], limits: Optional[dict] = None):
+    """
+    Run one dimension bayesian optimization
+
+    :param data: observed data
+    :param limits: optional limits for the search space
+    :return: 'mu, sigma, grid, suggestion_x'
+    """
+    # observed data
+    x_obs = np.array([[d["x"]] for d in data])
+    y_obs = np.array([d["y"] for d in data])
+
+    # create fake function black_box_function
+    black_box_function = lambda x: x
```

Comment:
Why?

Reply:
ill add it in the code. 

    # create fake function black_box_function
    # this function is not needed, but the package is requires this.
    # This is because the package is designed to find maximums of solve analytical equations

Reply:
this has now been removed